src/appleseed/foundation/meta/tests/test_aabb.cpp

```diff
@@ -396,13 +396,36 @@ TEST_SUITE(Foundation_Math_AABB)
         EXPECT_FEQ(sqrt(5.0 * 5.0 + 7.0 * 7.0 + 9.0 * 9.0), bbox.diameter());
     }
 
-    TEST_CASE(TestRadius)
+    TEST_CASE(TestSquareDiameter)
     {
         const AABB3d bbox(
             Vector3d(-1.0, -2.0, -3.0),
             Vector3d(4.0, 5.0, 6.0));
 
-        EXPECT_FEQ(sqrt(5.0 * 5.0 + 7.0 * 7.0 + 9.0 * 9.0) / 2.0, bbox.radius());
+        EXPECT_FEQ(5.0 * 5.0 + 7.0 * 7.0 + 9.0 * 9.0, bbox.square_diameter());
+    }
+
+    TEST_CASE(TestRadius)
+    {
+        const AABB3d bbox(
+            Vector3d(0.0, -2.0, -3.0),
```

Comment:
Since you haven't switched to particularly "easy" values, I suggest you stick to the same bbox as the other tests (with -1.0 instead of 0.0 for the min X value).